app/components/arclight/embed_component.html.erb

```diff
@@ -1,13 +1,10 @@
-<%= tag.p class: "media" do %>
-  <span class="media-body al-online-content-icon" aria-hidden="true"><%= helpers.blacklight_icon :online %></span>
-  <span class="col text-muted text-uppercase">
-    <%= t('arclight.views.show.embedded_content') %>
-  </span>
+<div id="embed">
+  <h2 class="al-show-sub-heading"><%= t('arclight.views.show.embedded_content') %></h2>
```

Comment:
do we for sure want to get rid of the "online content" icon?

Reply:
It's not in Gary's mockups, so my thought was that he didn't want it.

Reply:
i agree, but it seems different to make that decision at the level of arclight vs. just hiding or overriding it in NTA

Reply:
Yes, @jcoyne is correct that I didn't want the icon, but @thatbudakguy is correct that we probably shouldn't make that decision for everyone (my mockup was in an NTA ticket so it didn't occur to me to clarify that).

Reply:
Since this is for a new major release (1.0.0) and Gary is the designer of the original and because we're making many other visual changes, it seems like now is the ideal time to push in a change like this.

Reply:
Since this is for a new major release (1.0.0) and Gary is the designer of the original and because we're making many other visual changes, it seems like now is the ideal time to push in a change like this.

Reply:
It should be easy enough for downstream implementations to re-add an icon (e.g. in css).